contracts/gateway/BaseAmplifierGateway.sol

```diff
@@ -34,7 +34,7 @@ abstract contract BaseAmplifierGateway is IBaseAmplifierGateway {
      * This function is the entry point for general message passing between chains.
      * @param destinationChain The chain where the destination contract exists. A registered chain name on Axelar must be used here
      * @param destinationContractAddress The address of the contract to call on the destination chain
-     * @param payload The payload to be sent to the destination contract, usually representing an encoded function call with arguments
```

Comment:
Can you check the interface as well? `IAxelarGateway.sol`

Reply:
that one's clean, it's just this one that's inaccurate

https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/main/contracts/interfaces/IAxelarGateway.sol#L27-L38